Cubical/Core/PropositionalTruncation.agda

```diff
@@ -19,15 +19,20 @@ data ∥_∥ {ℓ} (A : Set ℓ) : Set ℓ where
   ∣_∣ : A → ∥ A ∥
   squash : ∀ (x y : ∥ A ∥) → x ≡ y
 
-recPropTrunc : ∀ {ℓ} {A : Set ℓ} {P : Set ℓ} → isProp P → (A → P) → ∥ A ∥ → P
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+
+recPropTrunc : ∀ {A : Set ℓ} {P : Set ℓ} → isProp P → (A → P) → ∥ A ∥ → P
```

Comment:
Why is A explicitly quantified here?

Reply:
Oh! I forgot about that. Will update soon

Reply:
No. For the same reason as mentioned above, we need to ensure that the level of `A` and `P` are the same variable.

Reply:
Tried to make them different and it doesn't cause any problem.
Will remove the explicit `A` here.